packages/strapi-plugin-i18n/services/content-types.js

```diff
@@ -72,9 +72,9 @@ const isLocalizedAttribute = (model, attributeName) => {
  * @returns {string[]}
  */
 const getNonLocalizedAttributes = model => {
-  return Object.keys(model.attributes)
-    .filter(attributeName => !['locale', 'localizations'].includes(attributeName))
-    .filter(attributeName => !isLocalizedAttribute(model, attributeName));
+  return intersection(keys(model.attributes), getVisibleAttributes(model)).filter(
```

Comment:
you don't even need the intersection nor the keys(model.attributes) you can do getVisibleAttributes(model).filter(...) directly